modules/userId/index.js

```diff
@@ -884,6 +935,16 @@ function updateInitializedSubmodules(dest, submodule) {
   }
 }
 
+function getConfiguredStorageTypes(config) {
+  return config?.storage?.type?.trim().split(/\s*&\s*/) || [];
```

Comment:
Any reason the storage.types isn't an array instead of string parsing? 

ie) ['cookie', 'html5'] instead of "cookie&html5"

Reply:
it could be an array, just thought about keeping the same `string` type in the documentation, an offer a value that's also a string. Since I am not seeing another option that supports more than one type.

Reply:
Makes sense, thank you for explaining